src/configuration/app-containers.md

```diff
@@ -116,11 +119,15 @@ The `type` can be:
 
 The `build` defines what will happen by default when building the application.
 
-It has a sub property `flavor` for which the possible values are:
+Its only property is `flavor`, which specifies a default set of build tasks to run. Flavors are language-specific.
+
+* PHP - default `composer`
+  * `composer` will run `composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader` if a `composer.json` file is detected.
+  * `drupal` will run `drush make` automatically if a `.make` file is detected.
+* Node.js
+  * `default` will run `npm prune --userconfig .npmrc && npm install --userconfig .npmrc` if a `package.json` file is detected. Note that this also allows you to provide a custom `.npmrc` file in the root of your application (as a sibling of the `.platform.app.yaml` file.)
 
-* `drupal` means that `drush make` will automatically run if you provide `.make` files.
-* `composer` means that `composer install` will automatically run if you provide a `composer.json` or `composer.lock` file.
-* `symfony` is an alisas for `composer`.
+In all languages you can also specify a flavor of `none` (which is the default for any language other than PHP and Node.js), which as the name suggests will take no action at all. That is useful when you want complete control over your build steps, such as to run a custom composer command or use an alternate Node.js package manager.
```

Comment:
It isn't the default. The default is `default`, even for python and ruby. In practice, they do the same as the `none` flavor though.